recipes/lenstool/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "lenstool" %}
+{% set version = "8.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://git-cral.univ-lyon1.fr/{{ name }}/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f14fb546a78ebbae3604739fe70089a98ca22668a85ba0d537fd9d63fc1c0786
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    - python
```

Comment:
This is normally not listed here. Why did you add it?

Reply:
I removed this statement from meta.yaml. 

Build passed, but raised 2 warnings related to exports. I guess it's a minor issue though.
<pre>
WARNING (lenstool): run-exports library package conda-forge::llvm-openmp-14.0.6-h61d9ccf_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (lenstool): interpreter (Python) package conda-forge::python-3.10.8-he7542f4_0_cpython in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
</pre>